packages/paste-core/components/side-panel/stories/components/SidePanelWithFilterContent.tsx

```diff
@@ -0,0 +1,115 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable import/no-extraneous-dependencies */
+import { Badge } from "@twilio-paste/badge";
+import { Box } from "@twilio-paste/box";
+import { Button } from "@twilio-paste/button";
+import { Disclosure, DisclosureContent, DisclosureHeading } from "@twilio-paste/disclosure";
+import { Heading } from "@twilio-paste/heading";
+import { Radio, RadioGroup } from "@twilio-paste/radio-group";
+import * as React from "react";
+
+import { SidePanel, SidePanelBody, SidePanelFooter, SidePanelHeader } from "../../src";
+
+export const SidePanelWithFilterContent: React.FC<React.PropsWithChildren> = () => {
+  return (
+    <SidePanel label="more filters side panel">
+      <SidePanelHeader>
+        <Heading as="h3" variant="heading30" marginBottom="space0">
+          More filters
+        </Heading>
+      </SidePanelHeader>
```

Comment:
suggestion(non-blocking): In this example the header doesn't have a border bottom like in the AI storybook. Is this what is in the design? With the filters almost always having scroll in the body it just looks a little strange without it.

![image](https://github.com/user-attachments/assets/569a56ac-d09b-4bcd-948f-e6c052322a64)
![image](https://github.com/user-attachments/assets/d235e701-38a6-4a17-bb8e-26b16c4f7741)

Reply:
yeah that's the design, I kind of agree with you though. cc @roxanagomez

Reply:
+1